docs/nidigital/class.rst

```diff
@@ -1262,87 +1203,74 @@ fetch_history_ram_cycle_pin_data
             :param site:
 
 
+                Site on which to retrieve History RAM data. Specify site as a string in the form of siteN,
+                where N is the site number. The VI returns an error if more than one site is specified.
+
                 
 
 
             :type site: str
             :param pin_list:
 
 
+                Pins for which to retrieve History RAM data. If empty, the pin list from the pattern
+                containing the start label is used. Call :py:meth:`nidigital.Session.get_pattern_pin_list` or :py:meth:`nidigital.Session.get_pattern_pin_indexes` with the start
+                label to retrieve the pins associated with the pattern burst.
+
                 
 
 
             :type pin_list: str
-            :param sample_index:
-
+            :param position:
 
-                
-
-
-            :type sample_index: int
-            :param dut_cycle_index:
 
+                Sample index from which to start fetching pattern information.
 
                 
 
 
-            :type dut_cycle_index: int
-
-            :rtype: tuple (expected_pin_states, actual_pin_states, per_pin_pass_fail)
-
-                WHERE
-
-                expected_pin_states (list of int): 
-
-
-                    
-
-
-                actual_pin_states (list of int): 
-
-
-                    
-
-
-                per_pin_pass_fail (list of bool): 
-
-
-                    
-
-
-
-fetch_history_ram_scan_cycle_number
```

Comment:
a bunch of methods went away, consider mentioning that in CHANGELOG.md. It can still be a single "changed" item.

Reply:
Had already update the changelog; added another item to better convey the changes.